src/cli/main.rs

```diff
@@ -44,6 +44,7 @@ where
                 Subcommand::Send(cmd) => cmd.run(rpc::Client::from(api)).await,
                 Subcommand::Info(cmd) => cmd.run(api).await,
                 Subcommand::Snapshot(cmd) => cmd.run(api).await,
+                #[cfg(feature = "attach")]
                 Subcommand::Attach(cmd) => cmd.run(api),
```

Comment:
Let's keep the command and print a user-friendly error message if it is used when compiled without the `attach` feature flag.

Reply:
Fixed.